include/SDL3/SDL_hints.h

```diff
@@ -726,7 +724,7 @@ extern "C" {
  *
  * The variable can be one of:
  *
- * - "#window": the javascript window object (default)
+ * - "#window": the javascript window object
```

Comment:
Did you mean to remove `(default)` here?

Reply:
Yes. The default technically applies to the window property now.

Reply:
It might be worth calling that out here. I'm on the fence whether this hint should override the property or instead be the default for the property. Is there a use case for users setting this to redirect an existing application?

Reply:
@icculus, do you have an opinion on this?

Reply:
Existing applications couldn't be using multiple windows on Emscripten since that doesn't work. Therefore, the hint would only apply to the 1 canvas element that exists. So, I don't think there would be a problem if it overrode the property since existing applications wouldn't be using properties anyway.